scenarios/csharp/dotnetcore/call-center/call-center/Program.cs

```diff
@@ -0,0 +1,408 @@
+
+            var displayName = $"call_center_{DateTime.Now.ToString()}";
+
+            var content_1 = new
+            {
+                displayName = displayName,
+                analysisInput = new
+                {
+                    conversations = new object[] {
+                        new
+                        {
+                            id = "conversation1",
+                            language = this.userConfig.language,
+                            modality = "text",
+                            conversationItems = conversationItems
+                        }
+                    }
+                },
+                tasks = new object[]
+                {
+                    new
+                    {
+                        taskName = "summary_1",
+                        kind = "ConversationalSummarizationTask",
+                        parameters = new
+                        {
+                            modelVersion = conversationSummaryModelVersion,
+                            summaryAspects = new string[]
+                            {
+                                "Issue",
+                                "Resolution"
+                            }
+                        }
+                    },
+                    new
+                    {
+                      kind = "ConversationalPIITask",
+                      taskName = "PII_1"
+                    }
+                }
+            };
+            var content_2 = System.Text.Json.JsonSerializer.Serialize(content_1);
+            var response = await RestHelper.SendPost(uri.Uri.ToString(), content_2, this.userConfig.languageSubscriptionKey, new HttpStatusCode[]{ HttpStatusCode.Accepted });
+            return response.Item1.Headers.GetValues("operation-location").First();
+        }
+
+        private async Task<bool> GetConversationAnalysisStatus(string conversationAnalysisUrl)
+        {
+            var response = await RestHelper.SendGet(conversationAnalysisUrl, this.userConfig.languageSubscriptionKey, new HttpStatusCode[]{ HttpStatusCode.OK });
+            using (JsonDocument document = JsonDocument.Parse(response.Item2))
+            {
+                return 0 == string.Compare("succeeded", document.RootElement.GetProperty("status").ToString(), StringComparison.InvariantCultureIgnoreCase);
+            }
+        }
+
+        private async Task WaitForConversationAnalysis(string conversationAnalysisUrl)
+        {
+            var done = false;
+            while (!done)
+            {
+                Console.WriteLine($"Waiting {waitSeconds} seconds for conversation analysis to complete.");
+                Thread.Sleep(waitSeconds * 1000);
+                done = await GetConversationAnalysisStatus(conversationAnalysisUrl);
+            }
+            Console.WriteLine();
+            return;
+        }
+
+        private async Task<((string, string)[], (string, string)[][])> GetConversationAnalysis(string conversationAnalysisUrl)
```

Comment:
>((string, string)[], (string, string)[][]) [](http://example.com/codeflow?start=27&length=42)

This is a sophisticated enough type that it merits strong representation -- the summary and analysis concepts could each have a class. "A tuple of arrays of tuples of two strings and a two-dimensional matrix of tuples of two strings" just doesn't flow off the tongue :D

Reply:
Thank you for your comments! I created a new type (ConversationAnalysisResult) for this.